datafusion/tests/sql.rs

```diff
@@ -3543,6 +3543,135 @@ async fn explain_analyze_runs_optimizers() {
     assert_contains!(actual, expected);
 }
 
+#[tokio::test]
+async fn tpch_explain_q10() -> Result<()> {
```

Comment:
Nice :+1: 
Maybe it would be good to have those in the tpch crate instead?
In that case we could include some / all other queries as well.

Reply:
I agree -- filed https://github.com/apache/arrow-datafusion/issues/1377 to track this